src/signals/incident-management/containers/Signaling/components/GraphDescription/GraphDescription.tsx

```diff
@@ -53,10 +53,10 @@ const GraphDescription: FunctionComponent<GraphDescriptionProps> = ({
   <Wrapper data-testid="graph-description">
     <DescriptionWrapper>
       <div>
-        <Title as="h3">{title}</Title>
-        <Paragraph>{description}</Paragraph>
+        <Title as="h3" data-testid="graph-title" >{title}</Title>
+        <Paragraph data-testid="graph-description">{description}</Paragraph>
```

Comment:
This id is not unique (see Wrapper above)

Reply:
Deleted duplicate